openhtf/plugs/__init__.py

```diff
@@ -115,6 +117,9 @@ class InvalidPlugError(Exception):
 
 class BasePlug(object): # pylint: disable=too-few-public-methods
   """All plug types must subclass this type."""
+  
+  def __init__(self):
+    self.logger = logging.getLogger(logs.RECORD_LOGGER)
```

Comment:
if you put this at the class level, it should still work and won't require a `super()` call